hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeysDeleteRequest.java

```diff
@@ -220,7 +220,6 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
       }
       break;
     case FAILURE:
-      omMetrics.decNumKeys(deleteKeys.size());
```

Comment:
// reset deleteKeys as request failed.      
deleteKeys = new ArrayList<>();

So, even without this fix also I don't see an issue. Have you observed any issue? 
Or you want to avoid unnecessary call to decNumKeys is this the reason for PR?

Reply:
Thanks for review, sorry for  i overlooked it in catch.
If in order to avoid unnecessary calls to decNumKeys, is it necessary to delete it?

Reply:
I will modify the title if necessary, otherwise I will close this pr.

Reply:
We can remove that unnecessary call. I am fine with it. You can update the Jira description.

Reply:
Thanks, PTAL!